include/system/analytics_manager.js

```diff
@@ -130,44 +140,54 @@ module.exports = function AnalyticsManagerModule(pb) {
      * @param {String} name
      * @return {Boolean} TRUE if was unregistered, FALSE if not found
      */
-    AnalyticsManager.unregisterProvider = function(name) {
-        if (!AnalyticsManager.isRegistered(name)) {
+    AnalyticsManager.unregisterProvider = function(name, site) {
+        if (!AnalyticsManager.isRegistered(name, site)) {
             return false;
         }
-        delete PROVIDER_HOOKS[name];
+        delete PROVIDER_HOOKS[site][name];
```

Comment:
In order to remain backward compatible we need to default site to global here if it is not provided.